pkg/rancher-desktop/window/index.ts

```diff
@@ -182,6 +182,46 @@ export function openMain() {
   app.dock?.show();
 }
 
+let view: Electron.BrowserView;
+
+export function openExtension(id: string) {
+  // const preloadPath = path.join(paths.resources, 'preload.js');
+  console.debug(`openExtension(${ id })`);
+
+  const window = getWindow('main') ?? undefined;
+
+  if (!window) {
+    return;
+  }
+
+  const windowSize = window?.getContentSize();
```

Comment:
The `?` should not be necessary (because if window was nullish we'd have returned already). This applies to all other uses in this method too.